src/main/java/com/senecafoundation/Scene/Scenario.java

```diff
@@ -1,37 +1,43 @@
 package com.senecafoundation.Scene;
 
-import java.util.Scanner;
+import java.util.List;
+import java.util.UUID;
 
-public class Scenario extends Choices
```

Comment:
Scenario does not inherit from Choices - it's not a child object that takes Choices and adds more functionality.  It does _depend on_  choices below.  So it's added as a property.